package.json

```diff
@@ -61,7 +61,7 @@
     "typescript": "^2.9.1"
   },
   "devDependencies": {
-    "eslint-config-twine": "github:TwinePlatform/eslint-config-twine#2.0.2",
+    "tslint-config-twine": "github:TwinePlatform/tslint-config-twine#2.1.0",
     "istanbul-merge": "^1.1.1",
```

Comment:
Is `istanbul-merge` being used? It looks from travis that we're just running two separate testing suites.